apps/chat/src/components/Chat/TalkTo/TalkToSlider.tsx

```diff
@@ -206,6 +200,16 @@ interface Props {
   onOpenMarketplaceTab: () => void;
 }
 
+const shouldRenderSlide = (
+  idx: number,
```

Comment:
use `index` word instead
```suggestion
  index: number,
```